lib/routemaster/resources/rest_resource.rb

```diff
@@ -18,8 +18,13 @@ def show(id=nil)
         @client.get(@url.gsub('{id}', id.to_s))
       end
 
-      def index
-        @client.get(@url)
+      def index(params: {}, filters: {})
+        params_and_filters = params.merge(filters)
+        if params_and_filters == {}
+          @client.get(@url)
+        else
+          @client.get(@url, params: filters.merge(params))
+        end
```

Comment:
1. I don't think we should be passing an empty hash for params just for the reason that we might want to change the defaults in the future.

2. I guess this way is more flexible in that it examines the response to determine if it is enumerable or not. If we have a situation where we perform a `get` on a collection rather than an index on the resource itself, then this method will still return an Enumerable response.